class-multisite.php

```diff
@@ -510,7 +510,7 @@ public function settings_tab()
         public function update_network_options()
         {
             if (!isset($_POST['rsssl_ms_nonce']) || !wp_verify_nonce($_POST['rsssl_ms_nonce'], 'rsssl_ms_settings_update')) return;
-            if (!current_user_can('manage_network_options')) return;
+            if (! rsssl_user_can_manage() ) return;
```

Comment:
Ik zou zelf iets consistenter zijn met code style. Soms kan je wel eens iets missen natuurlijk. 

Zelf gebruik heb ik een code style ingesteld in PHPStorm en wanneer ik een stuk code selecteer en cmd + optn + L klik dan wordt het geformat volgens de WordPress guidelines. 

Zie dit voor meer informatie: https://www.jetbrains.com/help/phpstorm/wordpress-aware-coding-assistance.html#configuring-wordpress-code-style

Reply:
@hesseldj klopt idd. Heb die functie verplaatst. Vandaar dat ie 1x verwijderd is. Ga even kijken naar de code stijl. Thanks